### ISSUE TITLE
Like or Dislike unavailable when connecting to Spotify Connect

### ISSUE THREAD

**Issue by Kia0ra** [plietar/librespot#254]
The Like or Dislike commands disappear from the player as soon as the Spotify Connect (using librespot/raspotify) device is chosen to play audio. Instead the Random / Repeat commands appear.

**Comment by derWeihnachtsmann** [comment]
Yes this is the default behaviour. If you want Like/dislike feature, you are free to write some lines in the evening hours and post a Pull request. xD

**Comment by Kia0ra** [comment]
Default behavior of other Spotify player (PC App, Spotify Web Player, Chromecast, etc) is to put the thumb up/thumb down commands for radio playing when you can discover new artists and to put the Random/Repeat commands when you play a chosen artist.

Ahah, yeah it will need me quite a few evening hours to wrote this particular lines ;)
